src/include/verbose.h

```diff
@@ -109,6 +109,18 @@ struct __wt_verbose_multi_category {
 /* Check if a given verbosity level satisfies the verbosity level of a category. */
 #define WT_VERBOSE_LEVEL_ISSET(session, category, level) (level <= S2C(session)->verbose[category])
 
+/* Check if a given verbosity level is met for eviction related messages. */
```

Comment:
We do not need these defines. It is not in the performance path to check whether the verbose levels are already set, it is okay to set if already set.